addon-test-support/trigger-file-select.js

```diff
@@ -0,0 +1,12 @@
+import { triggerEvent } from './trigger-event';
+
+/*
+  @method triggerFileSelect
+  @param {String|HTMLElement} selector
+  @param {Array} array of flies
+  @return {RSVP.Promise}
+  @public
+*/
+export function triggerFileSelect(selector, files = []) {
+  return triggerEvent(selector, 'change', files);
```

Comment:
my only issue with putting all the logic there rather then with `fire-event` is that then I would need to recreate the logic from the `fireEvent` and `buildBasicEvent` methods.